wx/lib/floatcanvas/FloatCanvas.py

```diff
@@ -601,7 +601,7 @@ def Draw(self, Force=False):
 
         """
 
-        if N.sometrue(self.PanelSize <= 2 ):
+        if any(self.PanelSize <= 2 ):
```

Comment:
The better would be `N.any`

Reply:
Thanks. It looks like I have put too much trust in ruff. Fixed it and created an issue for ruff: astral-sh/ruff#12416